rails/benchmarks/bm_actioncontroller_api.rb

```diff
@@ -0,0 +1,16 @@
+require 'bundler/setup'
+require 'action_controller'
+require_relative 'support/benchmark_rails'
+
+class ApiController < ActionController::API
+  def show
+    render json: "JSON", status: :ok
+  end
+end
+app = ApiController.action(:show)
+request = Rack::MockRequest.new(app)
+Benchmark.rails("actioncontroller/api", time: 5) do
+  response = request.get("/")
+  fail "Bad response #{response.status}" if response.status != 200
+  fail "Bad response #{response.body}" if response.body != "JSON"
```

Comment:
Obviously there's more tests that we could add, this should be a good starting point, no?